package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "axios": "^0.16.2",
     "dotenv": "^4.0.0",
+    "lodash.flatmap": "^4.5.0",
```

Comment:
Why not adding lodash instead of lodash.flatmap ?

Reply:
Because I'm only using flatmap